recipes/libcf/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libcf" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-beta11.tar.gz
+  url: http://uvcdat.llnl.gov/cdat/resources/{{ name }}-{{ version }}-beta11.tar.gz
+  sha256: e40d3826ca2b47538b075f69f716798e255fe396fd3ae68e8b051be57a951bb4
+
+build:
+  number: 1
+  skip: True  # [win or py3k] 
+
+requirements:
+  build:
+    - hdf5
+    - libnetcdf 
```

Comment:
`pin` according to the docs. We need that to ensure ABI stability and package compatibility.

The pinnings are automatically updated when we update the dependencies.